ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -283,13 +435,95 @@ namespace Ikarus {
           const auto u = uFunction.evaluate(quadPos);
 
           // Value of the Neumann data at the current position
-          const auto neumannValue
-              = neumannBoundaryLoad(toEigen(intersection.geometry().global(curQuad.position())), lambda);
+          const auto neumannValue = neumannBoundaryLoad(intersection.geometry().global(curQuad.position()), lambda);
           energy -= neumannValue.dot(u) * curQuad.weight() * intElement;
         }
       }
-
       return energy;
     }
+
+    template <typename ScalarType>
+    Eigen::Matrix<ScalarType, 3, 3> kgBending(const Eigen::Matrix<ScalarType, 2, 3> &jcur,
```

Comment:
may be this can also be wrapped in a file `bendingstrains.hh` similar to `membranestrains.hh`, but I do not know how difficult it is

Reply:
I also thought about this but this would lead to too much slow down, for a generic interface. Maybe we can unify in the future

Reply:
Okay. Also, may be, at least in terms of locking, usually membrane strains are of interest to be modified and hence a separate interface for it is useful than for the bending strains. We can think about this again when we also include stress-based implementations at some point and refactor shell implementations